js/interface-lists.js

```diff
@@ -112,11 +115,14 @@ function initUserFilePickerProvider(userFolder) {
     widgetData.userFolder = userFolder;
 
     var itemProvider = _.find(filePickerPromises, { id: userFolder.folder.provId });
+
+    // eslint-disable-next-line no-unused-vars
```

Comment:
@tonytlwu I've removed both those lines and it seems that everything is working, should I remove it?

Reply:
You're taking a promise object from the `filePickerPromises` array and then setting the promise to `null`. I think that's important before the object is removed from `filePickerPromises` in the next line. The promise object is then removed in the line after that. I think you should be able to just remove it in one go without needing to null-ify it first.

I tried it in https://jsfiddle.net/tonytlwu/fdnksjyc/ and found that setting to `null` doesn't do anything (I was hoping it would cancel the promise).